src/shared-components/button/index.js

```diff
@@ -44,6 +46,7 @@ class Button extends React.Component {
   static defaultProps = {
     disabled: false,
     buttonType: 'primary',
+    color: 'purple',
```

Comment:
ok so dont hate me for this but as I pointed out above I think it would be more appropriate if we pass a COLORS constant as prop value it is safer type-wise and make more consistent with our Radiance usage (for example Icons, callout, etc we pass the hex constant value or one of COLORS constants)

Reply:
haha no dont worry, i dont hate you 😄 i get the idea of the switch over. I made the change for now but as mentioned above, would love to talk more about this for future stuff.